paasta_tools/kubernetes_tools.py

```diff
@@ -310,6 +310,10 @@ def _set_disrupted_pods(self: Any, disrupted_pods: Mapping[str, datetime]) -> No
         "paasta.yelp.com/scrape_uwsgi_prometheus": str,
         "paasta.yelp.com/scrape_piscina_prometheus": str,
         "paasta.yelp.com/service": str,
+        "paasta.yelp.com/pool": str,
```

Comment:
can you use the `paasta_prefixed` function here too?

Reply:
can do!

Reply:
went ahead and used paasta_prefixed everywhere as you mentioned in your approval comment